eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/pom.xml

```diff
@@ -13,7 +13,7 @@
     <artifactId>spring-cloud-azure-sample-stream-eventhubs-binder</artifactId>
     <groupId>com.azure.spring</groupId>
     <version>1.0.0</version>
-    <name>Spring Cloud Azure Sample Stream Event Hubs Binder</name>
+    <name>Communicate to a Single Event Hubs Namespace via Spring Cloud Stream Framework</name>
```

Comment:
Same here. When should we use upper case?

Reply:
this is title format